api/catalog/api/controllers/elasticsearch/related.py

```diff
@@ -0,0 +1,42 @@
+from elasticsearch_dsl import Search
+
+from catalog.api.controllers.elasticsearch.utils import (
+    _exclude_filtered_providers,
+    _get_query_slice,
+    _get_result_and_page_count,
+    _post_process_results,
+)
+
+
+def related_media(uuid, index, filter_dead):
+    """
+    Given a UUID, find related search results.
+    """
+    search_client = Search(using="default", index=index)
+
+    # Convert UUID to sequential ID.
+    item = search_client
+    item = item.query("match", identifier=uuid)
```

Comment:
Is this variable declaration necessary?
```suggestion
    item = search_client.query("match", identifier=uuid)
```

Reply:
Good catch, fixed in 64a2ed9.